<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VariableInput</class>
 <widget class="QWidget" name="VariableInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>47</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VariableInput</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="lineEditName">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditMin">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditMax">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditElements">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditUnits">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxAxes">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>0</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <item>
      <property name="text">
       <string>x-axis</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>y-axis</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>point</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="horizontalSliderPoint">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelPoint">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonDelete">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
